src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -64,12 +64,12 @@ static inline void MPIDI_OFI_idata_set_error_bits(uint64_t * data_field, MPIR_Er
 }
 
 /* Get the error flag from the OFI data field. */
-static inline void MPIDI_OFI_idata_get_error_bits(uint64_t idata, int *err_res)
+static inline int MPIDI_OFI_idata_get_error_bits(uint64_t idata)
```

Comment:
Maybe I should have pointed this out on the commit that added these APIs. Because now we are adding them then immediately changing them when using.

Reply:
🙄 Fixed. Sorry.